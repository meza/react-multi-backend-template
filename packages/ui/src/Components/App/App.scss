@import '../../stylesheets/base/colours';

.App {
  background-color: $primaryBackgroundColour;
  color: $primaryColour;
  height: 100%;
  width: 100%;
}
