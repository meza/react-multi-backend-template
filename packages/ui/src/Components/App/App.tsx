import React from 'react';
import styles from './App.scss';

export const App = () => {

  return (
    <div className={styles.App}>
      Hello World
    </div>
  );
};

export default App;
