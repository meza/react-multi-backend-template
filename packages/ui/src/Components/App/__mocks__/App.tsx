import getMockComponent from '../../../../testUtils/getComponentMock';

export default getMockComponent('App');
