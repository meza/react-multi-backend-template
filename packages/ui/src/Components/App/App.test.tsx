import TestRenderer, { act, ReactTestRenderer } from 'react-test-renderer';
import { App } from './App';
import React from 'react';

describe('The app component', () => {

  test('it renders consistently', async () => {
    let component: ReactTestRenderer | null = null;

    await act(async () => {
      component = TestRenderer.create(<App/>);
    });

    expect(component).not.toBeNull();

    if (component) {
      expect((component as ReactTestRenderer).toJSON()).toMatchSnapshot();
    }
  });
});
