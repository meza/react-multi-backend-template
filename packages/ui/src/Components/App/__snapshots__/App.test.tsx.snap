// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The app component it renders consistently 1`] = `
<div
  className="App"
>
  Hello World
</div>
`;
